src/auth0/handlers/guardianPolicies.js

```diff
@@ -23,11 +26,12 @@ export default class GuardianPoliciesHandler extends DefaultHandler {
   async getType() {
     // in case client version does not support the operation
     if (!this.client.guardian || typeof this.client.guardian.getPolicies !== 'function') {
-      return null;
+      return {};
     }
 
     if (this.existing) return this.existing;
-    this.existing = await this.client.guardian.getPolicies();
+    const policies = await this.client.guardian.getPolicies();
```

Comment:
`getPolicies()` still returns an array of strings, but we "normalize" it before returning the result (modelling it as an object with a `policies` attribute).

Using an object makes it more uniform with other handlers in this project.